### PR TITLE
MP JWT 1.2: Add support for token header and token cookie MP Config props

### PR DESCRIPTION
**Pull request by ayoho**
For issue #13571
For issue #13572

### CODE REVIEW

File: dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -69,4 +69,16 @@ sharedKey.desc=Specifies the string that will be used to generate the shared key
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* or javax* system properties.
 
+# Do not translate "MicroProfile", "JWT", "MP-JWT", "mp.jwt.token.header", or "MicroProfile Config"
+tokenHeader=Token header
+tokenHeader.desc=The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties:74]: Change,
"The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured."
to,
"The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and beyond of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured."

File: dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -69,4 +69,16 @@ sharedKey.desc=Specifies the string that will be used to generate the shared key
...
 
+tokenHeader.Authorization=Authorization
+tokenHeader.Authorization.desc=Authorization header
+
+tokenHeader.Cookie=Cookie
+tokenHeader.Cookie.desc=Cookie header
+
+# Do not translate "MicroProfile", "JWT", "MP-JWT", "mp.jwt.token.cookie", or "MicroProfile Config"
+cookieName=Cookie name
+cookieName.desc=The name of the cookie that is expected to contain a MicroProfile JWT. The default value is Bearer. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.cookie MicroProfile Config property if one is configured. This value will be ignored unless tokenHeader or the mp.jwt.token.header MicroProfile Config property is set to Cookie.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties:84]: Change,
"The name of the cookie that is expected to contain a MicroProfile JWT. The default value is Bearer. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.cookie MicroProfile Config property if one is configured. This value will be ignored unless tokenHeader or the mp.jwt.token.header MicroProfile Config property is set to Cookie."
to,
"The name of the cookie that is expected to contain a MicroProfile JWT. The default value is Bearer. This attribute is only used by versions 1.2 and beyond of the MP-JWT feature. This value overrides the mp.jwt.token.cookie MicroProfile Config property if one is configured. This value is ignored unless tokenHeader or the mp.jwt.token.header MicroProfile Config property is set to Cookie."

**Comment by skasund** [comment]
L2 message review completed

**Comment by ayoho** [comment]
#run-libby-bot
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt_fat_tck,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.oidc.client_fat,com.ibm.ws.security.oidc.client_fat.config,com.ibm.ws.security.oidc.client_fat.e2e,com.ibm.ws.security.oidc.client_fat.jaxrs,com.ibm.ws.security.oidc.client_fat.saml,com.ibm.ws.security.oidc.server_fat,com.ibm.ws.security.oidc.server_fat.app-passwords,com.ibm.ws.security.oidc.server_fat.app-passwords.DerbyDB.jwt,com.ibm.ws.security.oidc.server_fat.app-passwords.DerbyDB.opaque,com.ibm.ws.security.oidc.server_fat.app-passwords.LocalStore.jwt,com.ibm.ws.security.oidc.server_fat.app-passwords.LocalStore.opaque,com.ibm.ws.security.oidc.server_fat.app-tokens,com.ibm.ws.security.oidc.server_fat.app-tokens.DerbyDB.jwt,com.ibm.ws.security.oidc.server_fat.app-tokens.DerbyDB.opaque,com.ibm.ws.security.oidc.server_fat.app-tokens.LocalStore.jwt,com.ibm.ws.security.oidc.server_fat.app-tokens.LocalStore.opaque,com.ibm.ws.security.oidc.server_fat.clientregistration,com.ibm.ws.security.oidc.server_fat.config,com.ibm.ws.security.oidc.server_fat.config.oauth,com.ibm.ws.security.oidc.server_fat.config.oidc,com.ibm.ws.security.oidc.server_fat.endpoint.authorize,com.ibm.ws.security.oidc.server_fat.endpoint.coveragemap,com.ibm.ws.security.oidc.server_fat.endpoint.discovery,com.ibm.ws.security.oidc.server_fat.endpoint.end_session,com.ibm.ws.security.oidc.server_fat.endpoint.introspect,com.ibm.ws.security.oidc.server_fat.endpoint.revoke,com.ibm.ws.security.oidc.server_fat.endpoint.token,com.ibm.ws.security.oidc.server_fat.endpoint.userinfo,com.ibm.ws.security.oidc.server_fat.feature,com.ibm.ws.security.oidc.server_fat.granttype.jwt,com.ibm.ws.security.oidc.server_fat.gui.tokenMgmt,com.ibm.ws.security.oidc.server_fat.jaxrs,com.ibm.ws.security.oidc.server_fat.jaxrs.config,com.ibm.ws.security.oidc.server_fat.oauth,com.ibm.ws.security.oidc.server_fat.oidc,com.ibm.ws.security.oidc.server_fat.saml,com.ibm.ws.security.saml.sso_fat,com.ibm.ws.security.saml.sso_fat.2,com.ibm.ws.security.saml.sso_fat.3,com.ibm.ws.security.saml.sso_fat.config,com.ibm.ws.security.saml.sso_fat.config.mapToUserRegistry,com.ibm.ws.security.saml.sso_fat.endpoint.samlmetadata,com.ibm.ws.security.saml.sso_fat.jaxrs,com.ibm.ws.security.saml.sso_fat.jaxrs.config,com.ibm.ws.security.saml.sso_fat.logout,com.ibm.ws.security.saml.sso_fat.logout.httpServletRequest,com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout,com.ibm.ws.security.saml.sso_fat.logout.IDP_initiated,com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.delegated,com.ibm.ws.security.social_fat.LibertyOP,com.ibm.ws.security.social_fat.multiProvider,com.ibm.ws.security.social_fat.okdServiceLogin

**Pull request merged**